### ISSUE TITLE
0001462:
Moving invitations: update

### ISSUE THREAD

**Issue by Gloirin** [tine20/Tine-2.0-Open-Source-Groupware-and-CRM#818]
**Reported by awieland on 11 Aug 2009 12:54**

**Version:** Leonie (2009-07) RC 1

After moving an invitation from the personal calendar to another one of my calendars, the personal calendar is not updated straight after closing the &quot;Edit Event&quot; window: the appointment is still visible.
Only after reload, the appointment is gone.

**Comment by Gloirin** [comment]
Related to #1564

**Comment by Gloirin** [comment]
**Comment posted by cweiss on 1 Feb 2010 13:50**

mhh, i always thought this is a feature, as it might be irritating when the event goes straight away...

you&#039;re shure we should change this?

**Comment by Gloirin** [comment]
**Comment posted by bbalazs on 16 Mar 2010 13:21**

Yeah definitely need to change this. Never show a no-more-existing view on a calendar and be consitent to similar events (like deleting / declining the appointment)

We could try to work with fade out. So the appointment is still visible after closing the dialogue, but then quickly fades out. This would need some finetuning on the times set for fading and is no must requirement.

**Comment by Gloirin** [comment]
**Comment posted by cweiss on 18 Apr 2010 12:29**

OK, 

checking scope in client is nothing we have yet -&gt; postpone

**Comment by Gloirin** [comment]
**Comment posted by cweiss on 23 May 2011 13:45**

this is the new gird behaviour ;-)
